packages/server/src/api/controllers/automation.ts

```diff
@@ -207,7 +210,7 @@ export async function find(ctx: UserCtx) {
   ctx.body = await db.get(ctx.params.id)
 }
 
-export async function destroy(ctx: UserCtx) {
+export async function destroy(ctx: UserCtx<void, DocumentDestroyResponse>) {
```

Comment:
Even if its identical, we would normally alias these very direct types to something specific to the controller, incase it is ever expanded/changed - e.g. `DeleteAutomationResponse`.